src/react/src/components/ContributeForm.jsx

```diff
@@ -94,6 +94,30 @@ class ContributeForm extends Component {
 
     handleUploadList = () => this.props.uploadList(this.fileInput);
 
+    containsTag = (content, tagName) => {
+        const tagRegex = new RegExp(`<${tagName}>.*?<\\/${tagName}>`, 'i');
+        return tagRegex.test(content);
+    };
+
+    parseContent = content => {
+        if (!this.containsTag(content, 'strong')) {
```

Comment:
Why you need to check tags here? Is this come from the backend?

Reply:
No. it's not. I check if a string contains tags not to proceed with function logic if it's no sens. Moved logic to separate function.

Reply:
Why string can contain tags?

Reply:
I added the tag to style List Name bold. It's requirements from the client.

Reply:
Where can I see this requirement? OSDEV-1145 doesn't contain any links to origin ticket.

Reply:
My bad https://opensupplyhub.atlassian.net/browse/OSDEV-1105. Added to PR description

Reply:
Ok, I see it now. Still, attempting to parse HTML using regular expression [considering as antipattern](https://stackoverflow.com/a/1732454). Please, find another solution that solve this issue and keep the codebase consistent at the same time.